app/models/game.rb

```diff
@@ -23,7 +23,6 @@ def reset_piece_players
     pieces.where(is_black: false).update_all(user_id: user_id_white)
   end
 
-  # rubocop:disable Metrics/AbcSize
```

Comment:
Nope, no need to do anything about it, you just have to add the comment back and it will stop complaining